flaky test à corriger

mailbox_manager/tests/api/mail_domain_accesses/test_api_mail_domain_accesses_delete.py::test_api_mail_domain__accesses_delete_owners_last_owner 

```-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED mailbox_manager/tests/api/mail_domain_accesses/test_api_mail_domain_accesses_delete.py::test_api_mail_domain__accesses_delete_owners_last_owner - assert 204 == 403
 +  where 204 = <Response status_code=204>.status_code
 +  and   403 = status.HTTP_403_FORBIDDEN
```

--- a/src/backend/mailbox_manager/factories.py
+++ b/src/backend/mailbox_manager/factories.py
@@ -26,7 +26,7 @@ class MailDomainFactory(factory.django.DjangoModelFactory):
         django_get_or_create = ("name",)
         skip_postgeneration_save = True
 
-    name = factory.Faker("domain_name")
+    name = factory.Sequence(lambda n: f"domain{n!s}.com")
     slug = factory.LazyAttribute(lambda o: slugify(o.name))
     support_email = factory.Faker("email")
 

